schemamd/render.go

```diff
@@ -225,6 +225,13 @@ nameLoop:
 			}
 		} else if childAtt, ok := block.Attributes[n]; ok {
 			for i, gf := range groupFilters {
+				if strings.ToLower(n) == "id" && childAtt.Description == "" {
+					if gf.topLevelTitle == "Read-Only" {
+						childAtt.Description = "The ID of this resource."
```

Comment:
I noticed you are not removing anymore the `if` [here](https://github.com/hashicorp/terraform-plugin-docs/pull/46/files#diff-e41a127c178c2eabe9f50f358b74b7c81df61ca60719d28f66a698009fd43b4cR73) that sets the description for `id`. What changed?

Reply:
Yes it's also necessary to remove that, and I'm not sure it's worth checking for empty description here anyway?

Reply:
If we don't do that, all descriptions currently set on `ID`s will be overridden by this change, making it impossible for users to tailor-make the description they want.

Reply:
But then only the description setting should be inside this condition, not the appending to Read-Only group too?